src/sentry/static/sentry/app/views/usageStats/usageStatsProjects.tsx

```diff
@@ -170,15 +183,17 @@ class UsageStatsProjects extends AsyncComponent<Props, State> {
   }
 
   getTableLink(project: Project) {
-    const {dataCategory, organization} = this.props;
-
-    if (dataCategory === DataCategory.TRANSACTIONS) {
-      return `/organizations/${organization.slug}/performance/?project=${project.id}`;
+    const {dataCategory, getNextLocations, organization} = this.props;
+    const {performance, projectDetail, issueList} = getNextLocations(project);
+
+    if (
+      dataCategory === DataCategory.TRANSACTIONS &&
+      organization.features.includes('performance-view')
+    ) {
+      return performance;
     }
 
-    return organization.features.includes('project-detail')
-      ? `/organizations/${organization.slug}/projects/${project.slug}/?project=${project.id}`
-      : `/organizations/${organization.slug}/issues/?project=${project.id}`;
+    return organization.features.includes('project-detail') ? projectDetail : issueList;
```

Comment:
This feature is always on now, and there are pull requests open to remove it. See #25337